components/dashboard/src/teams/TeamUsage.tsx

```diff
@@ -114,7 +114,7 @@ function TeamUsage() {
     const calculateTotalUsage = () => {
         let totalCredits = 0;
         billedUsage.forEach((session) => (totalCredits += session.credits));
-        return totalCredits.toFixed(2);
+        return totalCredits.toLocaleString(undefined, { maximumFractionDigits: 2 });
```

Comment:
```suggestion
        return totalCredits.toLocaleString(undefined, { minimumFractionDigits: 2, maximumFractionDigits: 2 });
```
This is what I wanted to get at: Money always has 2 digits (in our money system at least, which most humans are comfortable with).

Example code:
```
const n = 7777.52;
const n2 = 7777.5;

console.log(n.toLocaleString(undefined, { minimumFractionDigits: 2, maximumFractionDigits: 2 }));
console.log(n2.toLocaleString(undefined, { maximumFractionDigits: 2 }));
console.log(n2.toLocaleString(undefined, { minimumFractionDigits: 2, maximumFractionDigits: 2 }));
```
Output:
```
7,777.52
7,777.5
7,777.50
```

Reply:
/hold because I'd love to see this in as well @filiptronicek

Reply:
Agreed, let's do it!